packages/plugins/i18n/admin/src/translations/de.json

```diff
@@ -0,0 +1,64 @@
+  "CMEditViewCopyLocale.copy-text": "Von anderer Übersetzung übernehmen",
+  "CMEditViewCopyLocale.submit-text": "Ja, ausfüllen",
+  "CMListView.popover.display-locales.label": "Übersetzungen anzeigen",
+  "CheckboxConfirmation.Modal.body": "Möchtest du die Lokalisierung deaktivieren?",
+  "CheckboxConfirmation.Modal.button-confirm": "Ja, deaktivieren",
+  "CheckboxConfirmation.Modal.content": "Wenn Du die Lokalisierung deaktivierst, werden alle Ihre Inhalte mit Ausnahme des Inhalts, der Ihrem Standardgebietsschema zugeordnet ist (sofern vorhanden), gelöscht.",
+  "Field.localized": "Dieser Wert ist einzigartig für die ausgewählte Übersetzung",
+  "Field.not-localized": "Dieser Wert ist für alle Übersetzungen gleich",
+  "Settings.list.actions.add": "Neue Übersetzung hinzufügen",
+  "Settings.list.actions.delete": "Übersetzung löschen",
+  "Settings.list.actions.deleteAdditionalInfos": "Dies wird die aktive Übersetzung löschen.",
+  "Settings.list.actions.edit": "Übersetzung bearbeiten",
+  "Settings.list.description": "Konfiguriere die Einstellungen für das Internationalisierung Plugin",
+  "Settings.list.empty.description": "Das ist ein unerwartetes Verhalten. Möglicherweise hast du die Datenbank manuell modifiziert. Stelle sicher, dass mindestens eine Übersetzung in der Datenbank existiert, um Strapi nutzen zu können",
+  "Settings.list.empty.title": "Keine Übersetzungen vorhanden.",
+  "Settings.locales.default": "Standard",
+  "Settings.locales.list.sort.default": "Sortieren nach Standard-Übersetzung",
+  "Settings.locales.list.sort.displayName": "Sortieren nach Anzeigename",
+  "Settings.locales.list.sort.id": "Sortieren nach ID",
+  "Settings.locales.modal.advanced": "Fortgeschrittene Einstellungen",
+  "Settings.locales.modal.advanced.setAsDefault": "Als Standard-Übersetzung verwenden",
+  "Settings.locales.modal.advanced.setAsDefault.hint": "Es muss eine Standard-Übersetzung geben, ändere diese, indem eine andere Übersetzung als Standard-Übersetzung gewählt wird.",
+  "Settings.locales.modal.advanced.settings": "Einstellungen",
+  "Settings.locales.modal.base": "Grundeinstellungen",
+  "Settings.locales.modal.create.alreadyExist": "Diese Übersetzung existiert bereits",
+  "Settings.locales.modal.create.defaultLocales.loading": "Verfügbare Übersetzungen werden geladen...",
+  "Settings.locales.modal.create.success": "Übersetzung hinzugefügt",
+  "Settings.locales.modal.create.tab.label": "Wechseln zwischen den Grundeinstellungen und den fortgeschrittenen Einstellungen",
+  "Settings.locales.modal.delete.confirm": "Ja, löschen",
+  "Settings.locales.modal.delete.message": "Das Löschen dieser Übersetzung wird allen Inhalt in dieser Übersetzung löschen. Um Inhalte zu behalte, kopiere sie zuerst in andere Übersetzungen.",
+  "Settings.locales.modal.delete.secondMessage": "Übersetzung löschen?",
+  "Settings.locales.modal.delete.success": "Übersetzung gelöscht",
+  "Settings.locales.modal.edit.confirmation": "Fertig",
```

Comment:
```suggestion
  "Settings.locales.modal.edit.confirmation": "Abschließen",
```

Reply:
Are you sure "Abschließen" is in line with the rest of the dialog confirmation labels?